pkg/services/searchV2/bluge.go

```diff
@@ -249,8 +237,27 @@ var ngramQueryAnalyzer = &analysis.Analyzer{
 	},
 }
 
-func getNameNGramField(name string) bluge.Field {
-	return bluge.NewTextField(documentFieldNameNgram, name).WithAnalyzer(ngramIndexAnalyzer)
+// Names need to be indexed a few ways to support key features
+func newSearchDocument(uid string, name string, descr string, url string) *bluge.Document {
+	doc := bluge.NewDocument(uid)
+
+	if name != "" {
```

Comment:
should this return an error or skip this document? it seems like `name` is pretty important

Reply:
many panels do not have name :)   The UI just shows them as "_Missing title_"